# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kmymoney/uic.patch
# Copyright (C) 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/kadvancedbanksettingsdlgdecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/kadvancedbanksettingsdlgdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/kadvancedbanksettingsdlgdecl.ui	2005-06-13 20:59:43.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/kadvancedbanksettingsdlgdecl.ui	2005-12-16 22:17:58.000000000 +0000
@@ -353,8 +353,5 @@
 </connections>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/kbackupdlgdecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/kbackupdlgdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/kbackupdlgdecl.ui	2005-03-26 17:01:56.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/kbackupdlgdecl.ui	2005-12-16 22:18:05.000000000 +0000
@@ -207,8 +207,5 @@
 </customwidgets>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/kcurrencycalculatordecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/kcurrencycalculatordecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/kcurrencycalculatordecl.ui	2005-09-04 19:44:50.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/kcurrencycalculatordecl.ui	2005-12-16 22:10:15.000000000 +0000
@@ -379,7 +379,5 @@
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/kcurrencyeditdlgdecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/kcurrencyeditdlgdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/kcurrencyeditdlgdecl.ui	2005-06-13 20:59:43.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/kcurrencyeditdlgdecl.ui	2005-12-16 22:18:22.000000000 +0000
@@ -302,6 +302,5 @@
 </images>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kmymoneylineedit.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/keditequityentrydecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/keditequityentrydecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/keditequityentrydecl.ui	2005-06-13 20:59:43.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/keditequityentrydecl.ui	2005-12-16 22:11:13.000000000 +0000
@@ -304,7 +304,5 @@
 </images>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneypriceview.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/keditschedtransdlgdecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/keditschedtransdlgdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/keditschedtransdlgdecl.ui	2005-06-13 20:59:43.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/keditschedtransdlgdecl.ui	2005-12-16 22:11:41.000000000 +0000
@@ -856,18 +856,5 @@
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneyaccountcombo.h</includehint>
-    <includehint>kmymoneycombo.h</includehint>
-    <includehint>kmymoneycombo.h</includehint>
-    <includehint>kmymoneyaccountcombo.h</includehint>
-    <includehint>kmymoneypayee.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneycategory.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/kendingbalancedlgdecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/kendingbalancedlgdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/kendingbalancedlgdecl.ui	2005-09-09 19:54:48.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/kendingbalancedlgdecl.ui	2005-12-16 22:12:05.000000000 +0000
@@ -1061,18 +1061,5 @@
 </images>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyaccountselector.h</includehint>
-    <includehint>kmymoneyaccountselector.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneycategory.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneycategory.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/kenterscheduledialogdecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/kenterscheduledialogdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/kenterscheduledialogdecl.ui	2005-06-13 20:59:43.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/kenterscheduledialogdecl.ui	2005-12-16 22:18:37.000000000 +0000
@@ -656,11 +656,5 @@
 </connections>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kmymoneypayee.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneycategory.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneyaccountcombo.h</includehint>
-    <includehint>kmymoneyaccountcombo.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/kexportdlgdecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/kexportdlgdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/kexportdlgdecl.ui	2005-06-13 20:59:43.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/kexportdlgdecl.ui	2005-12-16 22:18:49.000000000 +0000
@@ -505,13 +505,5 @@
 </images>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kmymoneyaccountcombo.h</includehint>
-    <includehint>kcombobox.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/kfindtransactiondlgdecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/kfindtransactiondlgdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/kfindtransactiondlgdecl.ui	2005-06-13 20:59:43.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/kfindtransactiondlgdecl.ui	2005-12-16 22:12:24.000000000 +0000
@@ -1303,16 +1303,5 @@
 </images>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kmymoneyaccountselector.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyaccountselector.h</includehint>
-    <includehint>kmymoneylineedit.h</includehint>
-    <includehint>kmymoneylineedit.h</includehint>
-    <includehint>kmymoneylineedit.h</includehint>
-    <includehint>kmymoneyregistersearch.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/kimportdlgdecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/kimportdlgdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/kimportdlgdecl.ui	2005-06-13 20:59:44.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/kimportdlgdecl.ui	2005-12-16 22:19:06.000000000 +0000
@@ -387,12 +387,5 @@
 </customwidgets>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kcombobox.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kcombobox.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/kmymoneypricedlgdecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/kmymoneypricedlgdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/kmymoneypricedlgdecl.ui	2005-06-13 20:59:44.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/kmymoneypricedlgdecl.ui	2005-12-16 22:19:15.000000000 +0000
@@ -218,6 +218,5 @@
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kmymoneypriceview.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/knewaccountdlgdecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/knewaccountdlgdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/knewaccountdlgdecl.ui	2005-06-13 20:59:44.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/knewaccountdlgdecl.ui	2005-12-16 22:12:52.000000000 +0000
@@ -764,10 +764,5 @@
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneyequity.h</includehint>
-    <includehint>kmymoneycurrencyselector.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyaccountselector.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/knewaccountwizarddecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/knewaccountwizarddecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/knewaccountwizarddecl.ui	2005-09-04 19:32:48.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/knewaccountwizarddecl.ui	2005-12-16 22:19:29.000000000 +0000
@@ -1520,12 +1520,5 @@
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneycurrencyselector.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneypayee.h</includehint>
-    <includehint>kmymoneycombo.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/knewequityentrydecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/knewequityentrydecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/knewequityentrydecl.ui	2005-06-13 20:59:51.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/knewequityentrydecl.ui	2005-12-16 22:19:39.000000000 +0000
@@ -234,6 +234,5 @@
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kmymoneyedit.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/knewinvestmentwizarddecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/knewinvestmentwizarddecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/knewinvestmentwizarddecl.ui	2005-07-18 15:43:08.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/knewinvestmentwizarddecl.ui	2005-12-16 22:19:51.000000000 +0000
@@ -504,10 +504,5 @@
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kmymoneylineedit.h</includehint>
-    <includehint>kmymoneylineedit.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneycurrencyselector.h</includehint>
-    <includehint>kmymoneylineedit.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/knewloanwizarddecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/knewloanwizarddecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/knewloanwizarddecl.ui	2005-06-13 20:59:51.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/knewloanwizarddecl.ui	2005-12-16 22:20:12.000000000 +0000
@@ -5547,25 +5547,5 @@
 </images>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneylineedit.h</includehint>
-    <includehint>kmymoneypayee.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>knuminput.h</includehint>
-    <includehint>kmymoneycombo.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>knuminput.h</includehint>
-    <includehint>kmymoneycombo.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyedit.h</includehint>
-    <includehint>kmymoneyaccountselector.h</includehint>
-    <includehint>kmymoneyaccountselector.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneyaccountselector.h</includehint>
-    <includehint>kmymoneydateinput.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/kofxdirectconnectdlgdecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/kofxdirectconnectdlgdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/kofxdirectconnectdlgdecl.ui	2005-06-13 20:59:51.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/kofxdirectconnectdlgdecl.ui	2005-12-16 22:20:17.000000000 +0000
@@ -107,6 +107,5 @@
 </connections>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kprogress.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/konlinequoteconfigurationdecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/konlinequoteconfigurationdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/konlinequoteconfigurationdecl.ui	2005-06-13 20:59:51.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/konlinequoteconfigurationdecl.ui	2005-12-16 22:20:23.000000000 +0000
@@ -197,9 +197,5 @@
 </connections>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kcombobox.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/ksplitcorrectiondlg.ui kmymoney2-0.8.1/kmymoney2/dialogs/ksplitcorrectiondlg.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/ksplitcorrectiondlg.ui	2005-06-13 20:59:51.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/ksplitcorrectiondlg.ui	2005-12-16 22:20:38.000000000 +0000
@@ -230,7 +230,5 @@
 </customwidgets>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/ksplittransactiondlgdecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/ksplittransactiondlgdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/ksplittransactiondlgdecl.ui	2005-09-04 19:32:48.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/ksplittransactiondlgdecl.ui	2005-12-16 22:13:11.000000000 +0000
@@ -383,6 +383,5 @@
 </images>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kmymoneysplittable.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/kupdatestockpricedlgdecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/kupdatestockpricedlgdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/kupdatestockpricedlgdecl.ui	2005-06-13 20:59:51.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/kupdatestockpricedlgdecl.ui	2005-12-16 22:13:23.000000000 +0000
@@ -311,8 +311,5 @@
 </images>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kmymoneydateinput.h</includehint>
-    <includehint>kmymoneycurrencyselector.h</includehint>
-    <includehint>kmymoneysecurityselector.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/dialogs/mymoneyqifprofileeditordecl.ui kmymoney2-0.8.1/kmymoney2/dialogs/mymoneyqifprofileeditordecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/dialogs/mymoneyqifprofileeditordecl.ui	2005-06-13 20:59:52.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/dialogs/mymoneyqifprofileeditordecl.ui	2005-12-16 22:20:53.000000000 +0000
@@ -872,20 +872,5 @@
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kcombobox.h</includehint>
-    <includehint>kcombobox.h</includehint>
-    <includehint>kcombobox.h</includehint>
-    <includehint>kcombobox.h</includehint>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kurlrequester.h</includehint>
-    <includehint>klineedit.h</includehint>
-    <includehint>kpushbutton.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/views/kbankviewdecl.ui kmymoney2-0.8.1/kmymoney2/views/kbankviewdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/views/kbankviewdecl.ui	2005-06-13 20:59:52.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/views/kbankviewdecl.ui	2005-12-16 22:25:07.000000000 +0000
@@ -197,7 +197,5 @@
 </images>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kmymoneytitlelabel.h</includehint>
-    <includehint>kmymoneyaccounttree.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/views/kcategoriesviewdecl.ui kmymoney2-0.8.1/kmymoney2/views/kcategoriesviewdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/views/kcategoriesviewdecl.ui	2005-06-13 20:59:52.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/views/kcategoriesviewdecl.ui	2005-12-16 22:25:13.000000000 +0000
@@ -166,7 +166,5 @@
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kmymoneyaccounttree.h</includehint>
-    <includehint>kmymoneytitlelabel.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/views/kinvestmentviewdecl.ui kmymoney2-0.8.1/kmymoney2/views/kinvestmentviewdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/views/kinvestmentviewdecl.ui	2005-09-04 19:36:10.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/views/kinvestmentviewdecl.ui	2005-12-16 22:25:19.000000000 +0000
@@ -174,7 +174,5 @@
 </images>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kmymoneyaccountcombo.h</includehint>
-    <includehint>kmymoneytitlelabel.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/views/kpayeesviewdecl.ui kmymoney2-0.8.1/kmymoney2/views/kpayeesviewdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/views/kpayeesviewdecl.ui	2005-06-13 20:59:52.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/views/kpayeesviewdecl.ui	2005-12-16 22:25:28.000000000 +0000
@@ -536,6 +536,5 @@
 </images>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kmymoneytitlelabel.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/views/kscheduledviewdecl.ui kmymoney2-0.8.1/kmymoney2/views/kscheduledviewdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/views/kscheduledviewdecl.ui	2005-06-13 20:59:52.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/views/kscheduledviewdecl.ui	2005-12-16 22:25:35.000000000 +0000
@@ -191,11 +191,5 @@
 </images>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>qwidget.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>kpushbutton.h</includehint>
-    <includehint>klistview.h</includehint>
-    <includehint>kmymoneyscheduledcalendar.h</includehint>
-    <includehint>kmymoneytitlelabel.h</includehint>
 </includehints>
 </UI>
diff -Nur kmymoney2-0.8.1-orig/kmymoney2/views/kstocktransactionviewdecl.ui kmymoney2-0.8.1/kmymoney2/views/kstocktransactionviewdecl.ui
--- kmymoney2-0.8.1-orig/kmymoney2/views/kstocktransactionviewdecl.ui	2005-06-13 20:59:52.000000000 +0000
+++ kmymoney2-0.8.1/kmymoney2/views/kstocktransactionviewdecl.ui	2005-12-16 22:26:43.000000000 +0000
@@ -63,6 +63,5 @@
 </widget>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>qwidget.h</includehint>
 </includehints>
 </UI>
